<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormIncluirFuncionario</class>
 <widget class="QWidget" name="FormIncluirFuncionario">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro Funcionario</string>
  </property>
  <property name="styleSheet">
   <string notr="true">text-transform: uppercase;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">
QWidget#widget_2{
background-color: rgb(3, 3, 3,150);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>880</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget{
text-transform: uppercase;
border:1px solid blue;
	background-color: rgb(231, 231, 231);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QFrame#frame_2{
color:red;
border: none;
background-color:#2408D4;}

QLabel{color: white;
background-color:transparent;
font: 75 15pt &quot;MS Shell Dlg 2&quot;;

}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lb_FormProdutos">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>cadastro funcionario</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QFrame
{background: #FFF;
border: none
}
QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}

QLabel{
text-transform: uppercase;
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>80</y>
              <width>71</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>codigo</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="tx_codigoFuncionario">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="tx_NomeFuncionario">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>100</y>
              <width>321</width>
              <height>31</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>nome</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>80</y>
              <width>201</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>nome funcionario</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>440</x>
              <y>80</y>
              <width>111</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>sobrenome</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="tx_Sobrenome">
            <property name="geometry">
             <rect>
              <x>440</x>
              <y>100</y>
              <width>261</width>
              <height>31</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>sobrenome</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="tx_Cpf">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>160</y>
              <width>211</width>
              <height>31</height>
             </rect>
            </property>
            <property name="maxLength">
             <number>11</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>cpf</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::VisualMoveStyle</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>140</y>
              <width>121</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>cpf</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>140</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>cidade</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="tx_cidade">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>160</y>
              <width>141</width>
              <height>31</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>cidade</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="tx_estado">
            <property name="geometry">
             <rect>
              <x>400</x>
              <y>160</y>
              <width>61</width>
              <height>31</height>
             </rect>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="maxLength">
             <number>2</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>estado</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>400</x>
              <y>140</y>
              <width>61</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>estado</string>
            </property>
           </widget>
           <widget class="QGroupBox" name="groupBox">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>218</width>
              <height>71</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>ativo</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="rd_Ativo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string>ativo</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rd_cancelado">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string>cancelado</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QDateEdit" name="dt_DataAniversario">
            <property name="geometry">
             <rect>
              <x>470</x>
              <y>160</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(208, 208, 208);</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QDateEdit" name="dt_DataCadastro">
            <property name="geometry">
             <rect>
              <x>640</x>
              <y>160</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(194, 194, 194);</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>470</x>
              <y>140</y>
              <width>161</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>data aniversario</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>640</x>
              <y>140</y>
              <width>151</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>data cadastro</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cb_login">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>220</y>
              <width>201</width>
              <height>31</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>200</y>
              <width>121</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>login</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="Bt_Salva">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF;
padding:15px 60px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
 }
QPushButton:hover{
background-color: #40a286
}</string>
              </property>
              <property name="text">
               <string>salvar (f2)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="Bt_Cancelar">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #1E87F0;
color: #FFF;
padding:15px 60px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
 }
QPushButton:hover{
background-color: #40a286
}</string>
              </property>
              <property name="text">
               <string>cancelar (esc)</string>
              </property>
              <property name="shortcut">
               <string>Esc</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
